plugins/notification-resources/src/components/BrowserNotificatator.svelte

```diff
@@ -85,14 +87,14 @@
     const sidebarObjectId = getSidebarObject()?._id
 
     if (_id && _id === sidebarObjectId) {
-      await client.remove(value)
+      if (me?.role !== AccountRole.ReadOnlyGuest) await client.remove(value)
       return
     }
 
     const locObjectId = await getObjectIdFromLocation(getCurrentResolvedLocation())
 
     if (_id && _id === locObjectId) {
-      await client.remove(value)
+      if (me?.role !== AccountRole.ReadOnlyGuest) await client.remove(value)
```

Comment:
Let's remove all of such changes, since we do not send transactions in readonly guest mode at all.